en/Developers/Build themes.md

```diff
@@ -0,0 +1,22 @@
+Learn how to customize the appearance of the Obsidian interface by building your own theme.
+
+For more resources on how to build your own themes, check out the following resources:
+
+- [for Theme Designers](https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/for+Theme+Designers)
+
+## Submit your theme
+
+When you're ready, you can share your theme with the Obsidian community by submitting it to the [[Marketplace]].
+
+For an in-depth guide on how to submit your plugin, refer to [How to add your theme to the community theme store](https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/Guides/How+to+add+your+theme+to+the+community+theme+store) on the Obsidian Hub.
+
+## Share your theme with the community
+
+Share what you've built in the `#appearance` channel on our Discord server, or in the Share & Showcase category on our forum.
+
+## Community and support
+
+If you get stuck, or if you're looking for feedback, [join the community](https://obsidian.md/community).
+
+- `#appearance` channel on Discord.
```

Comment:
there's also `appearance-dev` for more advanced topics

Reply:
Also just saw "#theme-dev". We should clarify the differences.